### ISSUE TITLE
The One Issue with the Binyan Paul

### ISSUE THREAD

**Issue by IsraelLand** [IAHLT/UD_Hebrew#64]
@amir-zeldes

Since we now aspire to include a more complete feature set for verbs and so on, this raises some issues & conflicts.
For Beinoni Paul, we have no choice but the PAAL for HebBinyan as of today.

AFAIK this is the only case of a strictly active binyan, which gets a passive voice (though some of us would prefer a different voice in some cases).

Regarding the lemma, for "אסור","רצוי", "עשוי" we're supposed to do "אסר""רצה","עשה" if it's PAAL. Not to mention זקק for זקוק.
This is problematic for "regular" Paul's as a whole, but I think this is exacerbated with this specific set of words - which are prone to be AUX. 
I'm really not sure these lemmas are really optimal, specifically usage wise (Surely עשוי of "done" but I don't think it's that transparent in הוא עשוי ללכת, similar to how "he might go" isn't really thought of in terms of might and power, though not *mights).
Alternatively, we can retain the lemmas as the Paul form (רצוי for רצוי), but this is another conflict with PAAL.

Finally, uniformity across POS's. As these can be ADJs as well as Verbs (Paul), and we would like to keep this nonbinary spectrum (some of us more than others :)), it seems a bit wrong to lemmatize עשוי once as עשה for verbs, once as עשוי for ADJ - the same word basically.

Especially since these cases of Paul are extra fluid, in this Semitic environment, we think introducing Paul would minimize conflicts, be more true to form, and overall get the job done better.

You guys are more than welcome to add anything else (Linking this on Slack).

Thanks!

**Comment by IsraelLand** [comment]
Just a quick addition, on top of reasons of lemma, binyan, voice, the uniformity aspect is a big deal for us, as designating these as Paul, when indeed Verb, would solve a lot of headaches.

**verbal only -** 
I mentioned זקוק which is one case, which I think **can't** be adj - and thus we encounter the set of Paul problems.
For עלול we see a different case where it **can** attach to an inf. (זקוק can't) - but it **can't** be an adj as well, **nor** an NP in עלול לישועה*.
**both adjectival & verbal -**
Then עשוי **can be** both adj, if we see "a well made steak" as the same word, and **attach** to an inf.
Finally רצוי is **similar** to עשוי, but **can't** get an infinitive

All in all, many different cases, which we don't care about Binyan when they're ADJ, but some must be Verb - designating all of them as this one Paul who does its own thing, would be beneficial by solving a bunch of problems, and I personally think truer to form.

**Comment by amir-zeldes** [comment]
OK, I see several different issues here.

1. The HebBinyan feature, which is not a universal feature, is based on traditional grammar. As a custom feature, we can basically do whatever we want with it, but I'd be reluctant to add a binyan pa'ul, since this is not a traditional binyan and therefore would be confusing to users who know traditional Hebrew grammar. I think a necessary criterion for being a binyan in Hebrew is having finite forms. Historically, huf'al and pu'al are not 'really' independent binyans, at least if you look at it from the comparative perspective (e.g. peeking at Arabic). They don't have their own infinitives, for example; however, they have a past and future form, and they have participles. This is not true for pual, which is just the beinoni paul equivalent of paal. For these reasons, I would not give pa'ul as a value for binyan - but that doesn't need to affect our other decisions, since it's a custom feature and therefore doesn't interact with the rest.
2. Some of the items on the list are not really VERBs - for example, I don't think zakuk is synchronically a form of a hypothetical VERB zakak. I think it's as much an ADJ as tsarix, and there is no possibility of introducing a by-agent (??zakuk al yedey...)
3. For forms like asur, I think it is sensible to consider them related to asar - are you saying you don't want to do this? You can even have by phrases, though of course impersonal uses with no agent are more common:

  * במהלך המסע הארוך נשדדה שיירת הכלולות פעם אחר פעם על אף שהדבר היה אסור על ידי החוק הסאלי
  * אם השימוש בריקשה אסור על ידי יהודי בשבת, הוא אסור גם אם הגוי עושה זאת

4. Finally about the question whether passive participles should generally be lemmatized to the base VERB although they are adjectival, this is a common question in UD and the guidelines are pretty clear (though lemmatization is more of a 'best practices' thing in UD, as it's hard to validate): if something is tagged as a VERB then it should be lemmatized to the VERB. You see this duality in English as well, where entertaining/ADJ is lemmatized to itself but as a VERB to entertain. This is because there are some diagnostics in English tagging to distinguish them, and both are possible with slightly different senses. The famous PTB guideline example is:

  * The duchess was entertaining/VERB|ADJ last night

With a corresponding difference in meaning.

**Comment by IsraelLand** [comment]
Thank you.
I'm very much for freedom to choose ADJ or Verb. For adjectives, we have no problem anyway,
but for verbs, the features seem to conflict one another, which directs us to the ADJ direction, often for no good reason,
That's what I thought paul, grouping all verb cases thus avoiding conflictions, would alleviate.

1. The thing is, it affects us mostly validation wise. For example, I don't see zakuk as a passive paul. But once I tag for Verb-PAAL, I can only tag for passive, and then I must tag by nsubj:pass - for something I don't really think is passive. This creates a chain reaction, where a non-universal feature really interferes with my ability to tag the most correct way as I see fit. Netanel asked, if indeed it's not a universal feature? It seems to be in our Arborator.
2. Right, they all behave differently. zakuk cannot get a by agent, nor can alul. You think zakuk is an adjective? האיש הזקוק? This sounds bad to me, the alternative being האיש הנזקק. In any case, Paul **verbs** act very different from one another, and I think PAUL would group them easily under one banner. As I said, this affects binyan, voice and lemma restrictions, which then have tagging side effects. We must tag for Voice=pass (on a PAAL). > No mid. >nsubj:pass. Oftentimes, we just go the ADJ route to avoid this, and I don't think this is advisable.
3. That's fair, lemma wise, it's just that it would render stuff like עלול to עלל or זקוק to זקק (if not adj.) This is not the case in HTB (in their AUX form at least), anyway. Any decision would be fine but would have its cost, either by fixing old data or distance of form to origin  - I don't think עלול's lemma is seen as עלל any more than I see could's lemma as can, nor is עלל productive.
4. Right. I meant more in the sense of עשוי ללכת as mentioned in 3, these "things" which I think are seen as far-away from their original morphology, but I see how it's not that different from any other adj-verb מארחת - אירח מארח for lemmas.

So all things considered, would you prefer PAAL for lemmas, or the more surface form? 
Pass. for PAAL? As something mandatory, i.e. mid. not allowed & Binyan, voice as something which **must** be tagged? 
Some of us don't see all PAULs as pass., while the binyan itself is weird as PAAL. If binyan isn't universal, we don't have to.

Again, I think introducing Paul would be better in terms of avoiding conflictions of pass for paal, lemmas of non-productive forms, and allow free voice choice.

ratzuy|lemma=ratzuy|Binyan=PAUL|Voice=whatever seems a no brainer, while
ratzuy|lemma=ratz**a**/**uy**|Binyan=**PAAL**|Voice=**Passive** seems like a whole bunch of conflictions

**Comment by amir-zeldes** [comment]
Hi and sorry for the slow reply:

1. `For example, I don't see zakuk as a passive paul` - no, as I wrote above I don't think it's PAUL either, but then why tag it as a verb at all? I mean, it's not passive of an existing active verb 'zakak', so I think it's right to tag it as an ADJ and then everything is OK, no? If we have to use `AUX` for it in context, then it would become a non-verbal auxiliary (which is possible), and then it would not get a binyan and have itself as a lemma (like "efshar", which has no binyan)
2. האיש הזקוק - first of all, not all ADJs can be used attributively (in English, "afraid" and other historical a-ADJ words are normatively avoided in attribute position). Second, it fits other ADJ tests such as comparison 'yoter zakuk'. But most importantly, as I wrote above there is no synchronic verb it corresponds to, so seeing it as a defective VERB seems unmotivated to me.
3. No, I agree, the lemma should be zakuk, alul etc.

So to sum up, I see the problem, I think the best way is this:

alul/a/ot/im|lemma=alul (no binyan, ADJ or AUX in context, usuall AUX)
zakuk|lemma=zakuk (again no binyan, this is probably usually not AUX?)

But:

ratzuy|lemma=ratza|Binyan=PAAL|Voice=Passive

Since in this case, there is actually a corresponding verb, and 'paul' is not a binyan in the traditional sense...

**Comment by IsraelLand** [comment]
Thanks!

1. Yeah, I don't see it as passive, but I do see it as Paul. I think ADJ is weird, but I seem to be in the minority; The afraid example is fair, and it's true other alternatives are not appealing either (i.e. defective zakak zakuk).

Thanks for clarifying, so to sum up - 
**lemmas are similar to form for ADJ or AUX**, so alul and asuy  סטייק עשוי היטב 'a well done steak' (as well as עשוי ללכת)

but **ratza** (actual PAAL) as lemma for **non-defective verbs**, so ratza and asa הדבר עשוי\רצוי על ידם 'the thing is made\wanted by them' (admittedly quite subtle differences)

I assume we wouldn't be tempted to go the verb-route for defective verbs, anyway, but weird cases do tend to pop up.
e.g. אפשר, which is not PAUL, but there's no real productive lemma but its form.

Somewhat related to אפשר and validation problems caused by such cases, we're for introducing some sort of misc feature to optionally mark PAULs. That way it's not an actual Binyan, and can be deleted later if unneeded,
But only when applied, it would allow us to hop over validation for problematic tokens (mostly voice, it seems), which many seem to be in the PAUL or modal "regions".

**Comment by amir-zeldes** [comment]
Yes, the top part is right, with the exception of AUX which are truly inflected form (so lemma of AUX yihiye is still haya)

> Somewhat related to אפשר and validation problems caused by such cases, we're for introducing some sort of misc feature to optionally mark PAULs. That way it's not an actual Binyan, and can be deleted later if unneeded,
But only when applied, it would allow us to hop over validation for problematic tokens (mostly voice, it seems), which many seem to be in the PAUL or modal "regions".

I'm not sure why you want to do this, but of course it is possible in principle. It sounds like what you want is kind of like an etymological feature - in other words, you agree that 'alul' is not really a passive of anything (so it shouldn't have `Voice` or a lemma other than alul), but you still want to mark it as pa'ul in terms of the mishkal, is that right? 

While there's nothing really speaking against it (as long as it's in MISC and called something other than HebBinyan), I don't see a strong reason to do this, since we're not systematically annotating mishkal or passive etymologies for things which are not synchronically a passive.

**Comment by IsraelLand** [comment]
Thanks.

Yeah I don't see alul, or other not-so-transparent constructs (pauls) or words (some modals) as **inherently** THIS voice, or THAT binyan.
To me, a bunch of what we do **now**, is etymological without real need. Is "yitachen" really lemma=nitachen? Sure, but it's meaningless transparently. Following nitachen, is it passive in any real way?
Is "hatnay nitan leshinuy" really transparently passive at this point (Nitan al yedey whom?)?

So regarding voice, Alul is definitely not a true passive, even though it's paul, but I'm not sure what other examples Omer had in mind.
regarding lemma and binyan, we tag binyan and lemma paal for pauls, but Netanel pointed out that if we go by 3rd.masc.past "rule" of lemmatization, we should actually get the nifal "hu asuy", lemma ne'esa, nifal.

At this point I'm really for the guidelines you proposed in general, but I still don't see the benefit in meaningless features for essentially frozen forms, especially when they're constraining us.
So it's **not** so much a matter of binyan=paul, more of a way to hop over validation constraints upon validation constraints of deprels, when those constraints are caused by unimportant features.
I was actually originally for excluding a bunch of verbtype=mod's from validation constraints, but Omer pointed out binyan Paul is more inclusive, with the later caveat that it would just be an optional misc feature.

When I tag nsubj:pass for "hatnay nitan leshinuy", just because I must do voice=passive, I don't think it's right.
This goes well with some validation relaxations we should implement after our recent ccomp/csubj issue, anyway.
So it's either a specific list excluding some modals & pauls from specific validations ("efshar" already is), 
or a more general misc=paul or misc=ModalSpecial. Or just letting minor features dictate our deprels, which I'm not for.

**Comment by amir-zeldes** [comment]
> Is "yitachen" really lemma=nitachen?

I think synchronically this is a deponent item, so I would not lemmatize it to something else, but I would also not list Binyan or Voice for it

> Is "hatnay nitan leshinuy" really transparently passive at this point (Nitan al yedey whom?)?

This is trickier, because there are genuine cases of "nitan" which are passive (ha-sefer nitan lanu al yedey ha-nasi), and the question is then do you want to diagnose on a case by case basis (with intermediate cases causing problems) or just decide that this is morphologically passive, so we annotate it as such. In any case, I don't think it's the same as yitaxen.

> When I tag nsubj:pass for "hatnay nitan leshinuy", just because I must do voice=passive, I don't think it's right.

It's internally consistent, and it's saying "this is the ostensible morphology" (rather than usage, pragmatic meaning, etc.). I don't think there's something to wrong with it, but speaking from a UD perspective, the only way around it is to say that there are two "nitan"s, one of which is a lexicalized ADJ which is no longer a passive participle. I know it's frustrating, but we get this all over the place, including in English (think "interested" - if there's 'in', there can be no 'by', so not passive, if there's 'by' then it's passive, and if there's neither...??? similarly 'surprised/surprising', etc. etc.), so it's not really unusual. I think of it as an epiphenomenon of grammaticalization being a cline, and UD annotation forcing a categorical view of a spectrum phenomenon.

I'm also fine with annotating paul somehow, just not as Binyan, due to the traditional semantics which that suggests.

BTW, I did add Binyan 'nitpael', which the academy seems to recognize as a distinct value and which we had annotated in the public Wiki dataset, so the validator initially complained about it:

https://universaldependencies.org/he/feat/HebBinyan.html

**Comment by IsraelLand** [comment]
> > Is "yitachen" really lemma=nitachen?
> 
> I think synchronically this is a deponent item, so I would not lemmatize it to something else, but I would also not list Binyan or Voice for it


Great. So we'd exclude yitachen (or as a part of a larger exception list)


> > Is "hatnay nitan leshinuy" really transparently passive at this point (Nitan al yedey whom?)?
> 
> This is trickier, because there are genuine cases of "nitan" which are passive (ha-sefer nitan lanu al yedey ha-nasi), and the question is then do you want to diagnose on a case by case basis (with intermediate cases causing problems) or just decide that this is morphologically passive, so we annotate it as such. In any case, I don't think it's the same as yitaxen.

No I agree it's different than yitaxen. But I think the impersonal, at least, is never passive (and I don't think anyone would object strongly) so that should be a sufficient difference, I guess (we make a difference between verb or adj for exact forms, and this is voice "alone"...) Nor am I sure about "hatnay nitan leshinuy", btw...



> > When I tag nsubj:pass for "hatnay nitan leshinuy", just because I must do voice=passive, I don't think it's right.
> 
> It's internally consistent, and it's saying "this is the ostensible morphology" (rather than usage, pragmatic meaning, etc.)

Yeah I see what you mean, so long as we follow the strict morphology and it's consistent further.
I still feel discomfort when tagging it "down the chain" as X, while consistent, when I don't see X as true for the first link in the chain to begin with (yitaxen\nitan limtzo themselves aren't passive to me). Even if by strict morphology they are.



> I'm also fine with annotating paul somehow, just not as Binyan, due to the traditional semantics which that suggests.

Great. 
I think at this point we can vote what would be better for all of us (excluding from voice\binyan validation - the word yitaxen\nitan\a more comprehensive list of modals? pauls?\pauls as a whole by way of an optional misc feature)

Thank you very much!

**Comment by NathanD38** [comment]
@amir-zeldes 

A certain Beinoni Paul, _savur_, has a complement content clause: 
"ani savur ki / she-ha-be'aya murkevet yoter".

If we tag it as `VERB`, with that optional feature, and let it have `lemma=סבר, HebBinyan=PAAL, Voice=Mid`,
the complement content clause can receive `ccomp`, based on the other thread ("hu hishtaxne'a/shunxa' ki...").

Alternatively, if we tag it as `ADJ`, could the content clause receive `ccomp` still?
We do allow certain `ADJ` to send `csubj` when they function as the root of a main (impesonal) clause.

By the same token, could they send `ccomp` when there's an explicit `nsubj`?

Although there is no semantic difference between the PAAL _savar_ and the derived PAUL _savur_,
the latter collapses the meaning of "sover svara" (cognate objects?) into a single form with an internal implicit object,
where the content clause is essentially an `acl` to an implicit svara.

**Comment by amir-zeldes** [comment]
ADJ can take `ccomp` in general, however in this case I think there is really no difference between 'sover' and 'savur', and the latter is what is called a deponent (defective) passive, i.e. passive in form only, but not in argument structure. In UD Latin such verbs are handled by using `Voice=Pass` but giving them regular arguments (obj, nsubj), see:

http://universal.grew.fr/?custom=627d48ac3741f

**Comment by NathanD38** [comment]
@amir-zeldes 
Thank you! 

Is this something that the UD validator and our specific validator can accept, a passive root with an obj complement?
(I myself differentiate between obj and ccomp, and don't see them as equivalent, so I specifically ask about obj.)

The Voice=Pass feature in Latin is merely to signify the morphological form with no reference to their semantics (which is mainly active***)?
 
Or can it also be used to signifiy semantics? 

Because it seems we're taking a somewhat mixed route in our practice here.

*** Deponent verbs - Zeus knows Smyth and LSJ had their fun share with those in Classical Greek.

**Comment by IsraelLand** [comment]
That seems like a really good use for the OptionalPaul misc feature, this is a Paul that isn't truly passive, so we can avoid tagging voice for it (or go for active, or stay with passive, if we want, but in any case leaving all options on the table).
For yitaxen, at least, we just avoid the choice altogether (from now on), not sure what's the best practice for savur.

Just to add, we'll indeed exclude yitaxen, nitan so they won't have a voice nor a binyan in these cases, 
as well as an optional misc feature for paul as mentioned.
Thank you

**Comment by amir-zeldes** [comment]
> Or can it also be used to signifiy semantics?

I agree UD is ambiguous about this, and I bet different languages differ in UD in whether any morphological passive is tagged for passive Voice, or only semantically passive ones. It really depends on how clear and how morphologically marked the category is in the language. I think Hebrew has fairly clear passivization morphology (but not always semantics), and the whole Binyan and other tagging categories relate more to the morphology. It's mainly frozen expressions that are no longer 'live' forms of a verb (e.g. 'savur') where this gets tricky, since for things like 'nirtsax' it's completely clear that they are morphologically, syntactically and semantically passive.

**Comment by amir-zeldes** [comment]
> For yitaxen, at least, we just avoid the choice altogether (from now on), not sure what's the best practice for savur.

My personal opinion is not to tag savur as a form of savar, and not to tag it as passive either - it is now a frozen form IMO, not the passive of savar (unless in some weird sentence where you actually find something like "de'a she-svura 'al yedey rabim" :)
